bindings/wasm/lib/bindings.ts

```diff
@@ -20,6 +21,15 @@ const callUtilsMethod = (method: __UtilsMethods__): any => {
     }
 };
 
+const getSecretManagerFromWallet = (
+    method: WalletMethodHandler,
+): SecretManagerMethodHandler => {
+    // TODO figure out why this one is extensible but client isnt
```

Comment:
Is client from wasm really not extensible or is it only in the nodejs binding?

Reply:
tbh i am not sure why, i jsut know that this fixes the issue. maybe it has always existed even. Ive already spend soo much time on it though, not sure its worth it to do more